mappings/net/minecraft/block/BlockFire.mapping

```diff
@@ -48,7 +48,7 @@ CLASS none/amr net/minecraft/block/BlockFire
 		ARG 0 state
 		ARG 1 view
 		ARG 2 pos
-	METHOD e ()V
+	METHOD e registerFlammables ()V
```

Comment:
This just calls `registerFlammable` a bunch and is called on startup..? More descriptive than `init`.